intro-to-R-tidyverse/03-intro_to_r_exercise.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Exercise: Getting comfortable with R'
+output:
+  html_notebook:
+    toc: yes
+    toc_float: yes
+---
+
+The goal of these exercises is to help you get comfortable with using R and R 
+notebooks. 
+
+We will create a play data set for an imaginary mouse experiment where you have 
+control and treatment groups of mice. 
+
+### Create a imaginary mouse experiment
+
+Use the empty chunk below to create a *character* vector with made up sample IDs 
```

Comment:
If you limit the length of the `exp_group` vector, then maybe you should limit the length of the `mouse_id` vector as well since it comes first

Reply:
I didn't intend to have any limits on the vector length itself, but the number of groups. Can you point out the wording you think suggests this and how I can re-word it to make this clear?